SBBDesignSystemMobileSwiftUI/Views/SBBTabView/TabBarEntry/TabBarEntryView.swift

```diff
@@ -33,10 +34,11 @@ public struct TabBarEntryView: View, Identifiable {
         - imageView: The image displayed in the tab bar
         - label: The label display in the tab bar
         - tag: The tag used to reference the bar
-        - customBackground: The backgroud color of the tab (circle)
-        - customForeground: The foreground color of the tab (circle)
+        - warningBackground: Whether the backgroud color of the tab (circle) is in the primary color (red or blue)
+        - badge: Whether to display a small badge at the top right of the tab
+        - badgeNumber: The number to display inside the badge
      */
-    public init(contentView: AnyView? = nil, imageView: Image? = nil, label: String? = nil, accessibilityLabel: String? = nil, tag: AnyHashable, customBackground: Color? = nil, customForeground: Color? = nil) {
+    public init(contentView: AnyView? = nil, imageView: Image? = nil, label: String? = nil, accessibilityLabel: String? = nil, tag: AnyHashable, warningBackground: Bool = false, badge: Bool = false, badgeNumber: Int? = nil) {
```

Comment:
Only Numbers are permitted as Badge View? On SBB Mobile, there is e.g. a checkmark as bagde. 
Would it make sense to allow any views, or is there something against this idea?

Reply:
thank you :-)